### ISSUE TITLE
Please add dynamic config loader by file extension

### ISSUE THREAD

**Issue by Kachit** [phalcon/cphalcon#11000]
For example:

``` php
/* @var Phalcon\Config\Adapter\{Extension} */
$config = Phalcon\Config\Loader::load('/path/to/configfile.{extension}');
```

**Comment by dschissler** [comment]
@sergeyklay Can this be added to 4.0 milestone?  Also change title.

**Comment by Jurigag** [comment]
Well i can update `Phalcon\Config\Factory::load` to accept just string.

**Comment by Jurigag** [comment]
Well i have this ready, but we don't have 3.3.x branch, and 4.0.x is not merged, i doubt i can add it to 3.2.x since it's new feature/enhancement really.

**Comment by sergeyklay** [comment]
Implemented in the `3.3.x` branch. Thank you for contributing and for helping us make Phalcon better.
